zipkin-ui/js/component_ui/trace.js

```diff
@@ -443,7 +448,11 @@ export default component(function trace() {
     this.on(document, 'uiZoomInSpans', this.zoomInSpans);
     this.on(document, 'uiZoomOutSpans', this.zoomOutSpans);
 
+    this.on(document, 'uiShowJson', this.showJson);
+
     const self = this;
+    self.raw = data.raw;
```

Comment:
ex not sure why the website doesn't render it, but here's the definition

- name: raw
in: query
required: false
description: |
Note this flag has no value. Ex. /trace/{traceId}?raw
Normally, the trace endpoint cleans trace data. For example, it merges
spans by id, adds missing timestamp or duration, corrects clock skew..
Specifying this flag is a debug case, when you are debugging zipkin
logic or zipkin instrumentation, and want to see the input to these
adjusters. For example, this might explain or rule out clock skew.
type: boolean

https://github.com/openzipkin/zipkin-api/blob/master/zipkin-api.yaml#L146